frontend/src/pages/article/TagList/Tag/Tag.styled.ts

```diff
@@ -7,7 +7,7 @@ export const TagBox = styled.li<{ isSelected: boolean }>`
   cursor: pointer;
 
   &:hover {
-    background-color: #007bff;
+    ${({ isSelected }) => !isSelected && `background-color: #49a0fcff;`}
```

Comment:
```suggestion
    ${({ isSelected }) => !isSelected && "background-color: #49a0fcff;"}
```
백틱이 아닌 큰따옴표로도 괜찮을 것 같네요!😮